libs/vkd3d/resource.c

```diff
@@ -4769,6 +4769,7 @@ bool vkd3d_create_texture_view(struct d3d12_device *device, const struct vkd3d_t
     object->vk_image_view = vk_view;
     object->format = format;
     object->info.texture.vk_view_type = desc->view_type;
+    object->info.texture.aspect_mask = desc->aspect_mask;
```

Comment:
Do we remember to hash the aspect mask anywhere?

Reply:
Only case I can think of is when creating 3 different R8_UNORM views of Y, Cb, and Cr of 420.

Reply:
Right, need to do that.